frontend/src/components/CommentView.js

```diff
@@ -231,6 +232,7 @@ const CommentView = ({ comment, allUsers, getComments }) => {
                 {comment.replies.map((reply) => {
                     return (
                         <ReplyView
+                            key={reply._id}
```

Comment:
this key should suffice? otherwise why are you also passing one on 113. only one should be needed